netlify/functions/newsletter_subscription.js

```diff
@@ -0,0 +1,42 @@
+const mailchimp = require("@mailchimp/mailchimp_marketing");
+const md5 = require("md5");
+
+exports.handler = async function (event) {
+    if (event.httpMethod == 'POST') {
+        const listId = "d5aaf678de";
```

Comment:
Do we need to hardcode this? I think a better logic would be to define an environment variable for this as well.

Reply:
Yes, it isn't any confidential information. We can surely hardcode it. Moreover, @derberg will update this with new list ID from AsyncAPI account.

Reply:
If this value isn't confidential and doesn't change periodically (which I think doesn't), we are good to go :)

Reply:
If this value isn't confidential and doesn't change periodically (which I think doesn't), we are good to go :)